code/modules/mob/language.dm

```diff
@@ -230,10 +230,10 @@
 	var/sounds = rand(2, 8)
 	var/i = 0
 	var/newname = ""
-
+	var/vox_name_syllables = list("ti","hi","ki","ya","ta","ha","ka","ya","chi","cha","kah")
```

Comment:
```suggestion
	var/list/vox_name_syllables = list("ti","hi","ki","ya","ta","ha","ka","ya","chi","cha","kah")
```

Reply:
I _swear_ that's what I did in my code. It wouldn't have worked otherwise. How the _heck_ did it pass Travis?

Reply:
Oh, it's just...Oh...huh.